### PR TITLE
Create form payment anhd withdrawal

### PR DESCRIPTION
**Pull request by binhpt-1239**
Related Tickets
• [#47137](https://edu-redmine.sun-asterisk.vn/issues/47137)
What
- Feature Payment & withdrawal for user

How
-Create form currencies with field amount and hidden field currencies_type_id
- Handle create in currencies_controller

Screenshot from 2022-03-21 17-34-06
Screenshot from 2022-03-21 17-18-14
Screenshot from 2022-03-21 17-17-57

### CODE REVIEW

**Comment by binhpt-1239** [comment]
ready

**Review by nvatuan — COMMENTED** [review]
OK

**Comment by BaoKhanh99** [comment]
ok

File: app/controllers/currencies_controller.rb [hunk 1]
```diff
@@ -1,8 +1,39 @@
...
   end
+
+  def new
+    @currency = current_user.currencies.build
+  end
+
+  def create
+    currency = current_user.currencies.new currency_params
+    if currency.save
+      flash[:success] = t "notification.currency.create.success"
```
Comment (huongnt-2545) [inline app/controllers/currencies_controller.rb:17]: tìm hiểu thằng I18n Lazy lookup đi em để gọi ra cho ngắn gọn

Comment (binhpt-1239) [reply app/controllers/currencies_controller.rb]: Em đã sửa chỗ này rồi ạ bằng I18n Lazy rồi ạ!

**Comment by binhpt-1239** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]
merged

**Pull request merged**